src/test/java/com/puppycrawl/tools/checkstyle/PackageNamesLoaderTest.java

```diff
@@ -234,7 +234,7 @@ public void testUnmodifiableCollection() throws Exception {
         final Set<String> actualPackageNames = PackageNamesLoader
                 .getPackageNames(new TestUrlsClassLoader(Collections.emptyEnumeration()));
 
-        assertThrows(UnsupportedOperationException.class,
+        getExpectedThrowable(UnsupportedOperationException.class,
                 () -> actualPackageNames.add("com.puppycrawl.tools.checkstyle.checks.modifier"));
```

Comment:
In issue we meant, use `get.....` to return value.
Now this code looks weird, we ignoring a get. 
Please update such cases to make variable and fo asser in next statement for message

Reply:
I will update by this weekend.

Reply:
Done